_pages/about/release_policy.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: static
+title: Release policy
+---
+
+# Release policy
+As said during the Forum PHP 2015, we should give our users a clear view of how/when we release atoum.
```

Comment:
for now I just copy what is inside https://github.com/atoum/atoum.github.io/issues/16 I have not  work on the for yet